src/driver_client/dclient.rs

```diff
@@ -115,6 +113,7 @@ impl DriverClient {
             CMS_ADDR::ADDR_CPU2HIF_CMS_INITIALIZE,
             1,
         )?;
+        sleep(Duration::from_millis(200));
```

Comment:
Is this 100% needed?
If not, this can be a significant amount of latency/delay added

Reply:
same question below, I close this one.